README.md

```diff
@@ -1,94 +1,79 @@
-<a name="module_google-maps-image-api"></a>
-#google-maps-image-api
-<a name="module_google-maps-image-api..BASE_URL"></a>
-##google-maps-image-api~BASE_URL
-Allows you to easily get Google Maps Image API urls.
+# google-maps-image-api
 
-Google Maps Image API allows you to get static images of a 2d map or streetview.
+**Allows you to easily get Google Maps Image API urls.**
 
-This module will return a string which is a url you can load an image from.
+Google Maps Image API allows you to get static images of a 2d map or streetview. This module will return the url of such an image. The options correlate to those in the API.
 
-Options passed in correlate to properties you can pass to Google Maps Image API.
+## usage
 
-Here are parameter's you can pass through opts to get a url:
-
-```javascript
-{
-	//// REQUIRED ////
-	type: 'staticmap', // This value is required if you want to render a static map
-	
-	center: '40.714728,-73.998672', // This will be the center of your image. 
-									// It can also be a street address
- 
+```js
+const image = require('google-maps-image-api')
+image(opts) // -> url
```

Comment:
It would be good to have a copy and pasteable example here. So just put in some value for `opts` which work.